week2/homework/src/index.js

```diff
@@ -1,3 +1,79 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+const { readFile, writeFile } = require('fs');
+const { promisify } = require('util');
+
+const reading = promisify(readFile);
+const writing = promisify(writeFile);
+const toDO = 'todolist.json';
+
+
+async function content() {
+    const [, , command, ...args] = process.argv;
+    const entry = args.join(' ');
+    switch (command) {
+        case ('add'): {
+            const currentList = await toRead();
+            currentList.push(entry);
+            if (!entry) { console.log('Please add todo after add command'); return args; }
```

Comment:
Ok but it makes more sense to do that check at the very beginning of your `'add'` case, since we don't need to read the list or to push the entry to the array if it is missing.